tests/pyrog/conftest.py

```diff
@@ -2,5 +2,15 @@
 
 from . import factories
 
-register(factories.TemplateFactory)
 register(factories.SourceFactory)
+register(factories.ResourceFactory)
```

Comment:
can you add a comment above to explain that `register` exposes pytest fixtures for `<model>_factory` and `<model>` that may be used in all tests ?